graalvm-native-image-demo/spring-boot-ot/run.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+JAVA="$HOME/tools/GRAALVM21"
```

Comment:
The line is correct? If users set `export GRAALVM_HOME=/root/graalvm-test/graalvm-17-23.0.2-ali-1.1-instru` according to README.md, it seems they can't locate to relevant directory.